iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/provisioning/ProvisioningServiceClientAndroidRunner.java

```diff
@@ -0,0 +1,19 @@
+/*
+ *  Copyright (c) Microsoft. All rights reserved.
+ *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+
+package com.microsoft.azure.sdk.iot.android.provisioning;
+
+import com.microsoft.azure.sdk.iot.android.helper.TestGroup1;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+
+import tests.integration.com.microsoft.azure.sdk.iot.provisioning.ProvisioningServiceClientTests;
+
+@TestGroup1
+public class ProvisioningServiceClientAndroidRunner extends ProvisioningServiceClientTests
+{
+
```

Comment:
Intentionally empty class. The important part of this file is that the class extends ```ProvisioningServiceClientTests``` and is assigned to a test group